cfpb/cfgov-refresh
# Convert breadcrumbs left icon to svg

Update the breadcrumbs icons to use the new svg. Note: it appears the design was updated to make the new icon thinner than the weight of the previous version that's on the live website.

## Additions

-

## Removals

-

## Changes
- replaced CSS pseudoelement to style the icon with SVG styles
- add svg template tag to the breadcrumbs markup to use the new svg icon 

## Testing

1. `npm install`
1. `gulp clean && gulp build`
1. Check out the breadcrumbs icon above the secondary navigation at http://localhost:8000/data-research/consumer-credit-trends/credit-cards/
1. Run `gulp test:unit`

## Screenshots

### Before
![screen shot 2018-05-16 at 11 24 45 am](https://user-images.githubusercontent.com/702526/40126655-c8672472-58fb-11e8-9522-43a60683d7bc.png)


### After
![screen shot 2018-05-15 at 6 22 04 pm](https://user-images.githubusercontent.com/702526/40126625-b6252a66-58fb-11e8-8ab3-6a5dac6e7a4e.png)


## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
